{"hierarchy":{"paths":[["doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK","doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSWebAuthnError"]]},"primaryContentSections":[{"declarations":[{"platforms":["iOS"],"tokens":[{"kind":"keyword","text":"case"},{"text":" ","kind":"text"},{"kind":"identifier","text":"`internal`"},{"kind":"text","text":"("},{"text":"Error","kind":"typeIdentifier","preciseIdentifier":"s:s5ErrorP"},{"text":"?)","kind":"text"}],"languages":["swift"]}],"kind":"declarations"}],"sections":[],"abstract":[{"type":"text","text":"The authorization attempt failed for an internal reason."}],"metadata":{"roleHeading":"Case","modules":[{"name":"TSAuthenticationSDK"}],"symbolKind":"case","externalID":"s:19TSAuthenticationSDK15TSWebAuthnErrorO8internalyACs0E0_pSgcACmF","title":"TSWebAuthnError.internal(_:)","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"`internal`","kind":"identifier"},{"text":"(","kind":"text"},{"text":"Error","preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier"},{"text":"?)","kind":"text"}],"role":"symbol"},"kind":"symbol","identifier":{"url":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSWebAuthnError\/internal(_:)","interfaceLanguage":"swift"},"schemaVersion":{"minor":3,"patch":0,"major":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/tsauthenticationsdk\/tswebauthnerror\/internal(_:)"]}],"references":{"doc://com.TransmitSecurity.TSAuthenticationSDK/documentation/TSAuthenticationSDK":{"title":"TSAuthenticationSDK","type":"topic","abstract":[],"kind":"symbol","identifier":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK","url":"\/documentation\/tsauthenticationsdk","role":"collection"},"doc://com.TransmitSecurity.TSAuthenticationSDK/documentation/TSAuthenticationSDK/TSWebAuthnError/internal(_:)":{"abstract":[{"text":"The authorization attempt failed for an internal reason.","type":"text"}],"title":"TSWebAuthnError.internal(_:)","identifier":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSWebAuthnError\/internal(_:)","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"`internal`","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:s5ErrorP","kind":"typeIdentifier","text":"Error"},{"text":"?)","kind":"text"}],"role":"symbol","kind":"symbol","type":"topic","url":"\/documentation\/tsauthenticationsdk\/tswebauthnerror\/internal(_:)"},"doc://com.TransmitSecurity.TSAuthenticationSDK/documentation/TSAuthenticationSDK/TSWebAuthnError":{"abstract":[],"title":"TSWebAuthnError","type":"topic","identifier":"doc:\/\/com.TransmitSecurity.TSAuthenticationSDK\/documentation\/TSAuthenticationSDK\/TSWebAuthnError","kind":"symbol","role":"symbol","url":"\/documentation\/tsauthenticationsdk\/tswebauthnerror","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"TSWebAuthnError","kind":"identifier"}],"navigatorTitle":[{"kind":"identifier","text":"TSWebAuthnError"}]}}}